jest.legacy.config.js

```diff
@@ -7,7 +7,9 @@ module.exports = {
   moduleFileExtensions: ["js", "json", "jsx", "ts", "tsx"],
   moduleNameMapper: {
     "^luxon$": "<rootDir>/node_modules/luxon",
-    "^react$": "<rootDir>/node_modules/react",
+    "^react$": "react-17",
+    "^react-dom$": "react-dom-17",
+    "^react-dom/test-utils$": "react-dom-17/test-utils",
```

Comment:
**question (non-blocking):** And here we use the alias name only, not the full `node_modules` path?

Reply:
Yup, when jest runs its `transform` over the code (via `babel-jest`) it will preprocess the code and swap out refs before execution.